qtconsole/frontend_widget.py

```diff
@@ -482,10 +482,8 @@ def _handle_execute_reply(self, msg):
             status = content['status']
             if status == 'ok':
                 self._process_execute_ok(msg)
-            elif status == 'error':
-                self._process_execute_error(msg)
-            elif status == 'aborted':
```

Comment:
https://github.com/ipython/ipykernel/issues/190 fixes the bug where KeyboardInterrupt is turned into status=aborted